doc/conf.py

```diff
@@ -302,3 +302,14 @@
     }
 
 numpydoc_class_members_toctree = False
+
+def touch_example_backreferences(app, what, name, obj, options, lines):
+    # generate empty examples files, so that we don't get
+    # inclusion errors if there are no examples for a class / module
+    examples_path = os.path.join(app.srcdir, "generated", "%s.examples" % name)
+    if not os.path.exists(examples_path):
+        # touch file
+        open(examples_path, 'w').close()
```

Comment:
See related issue:

https://github.com/sphinx-gallery/sphinx-gallery/issues/126

And the fix that I committed that I think should make this unnecessary:

https://github.com/mne-tools/mne-python/commit/97b38c9a75c06ffaecf5eef588a689899f86ded2#diff-684a0ceb2878d48b19612ca360e277efR297